internal/controllers/database/init.go

```diff
@@ -62,9 +63,61 @@ func (r *Reconciler) setInitDatabaseCompleted(
 		Reason:  ReasonCompleted,
 		Message: message,
 	})
+	meta.SetStatusCondition(&database.Status.Conditions, metav1.Condition{
+		Type:   DatabaseWaitOperationCondition,
+		Status: metav1.ConditionFalse,
```

Comment:
Maybe invert logic and set `Status` to `True` for operation condition? As a user, I expect that all conditions for object should be true. Status `False` in some condition for healthy Database is implicitly